src/omotes_sdk/internal/orchestrator/orchestrator_interface.py

```diff
@@ -155,3 +191,22 @@ def send_job_result(self, job: Job, result: JobResult) -> None:
         self.broker_if.send_message_to(
             OmotesQueueNames.job_results_queue_name(job), result.SerializeToString()
         )
+
+    def request_workflows_handler(self, request_workflows: RequestAvailableWorkflows) -> None:
+        """When an available work flows request is received from the SDK.
+
+        :param request_workflows: Request available work flows.
+        """
+        logger.info("Received an available workflows request")
+        self.send_available_workflows(self.workflow_type_manager)
+
+    def send_available_workflows(self, workflow_type_manager: WorkflowTypeManager) -> None:
```

Comment:
Why is `workflow_type_manager` received as an argument instead of referenced through `self.workflow_type_manager`? Both instances where `send_available_workflows` are used, the argument is the same.

Reply:
This one was resolved but I didn't see a reply to the question. Perhaps this one was missed?

Reply:
Yes it was missed :-)